xbmc/video/windows/GUIWindowVideoBase.h

```diff
@@ -33,8 +33,7 @@ class CGUIWindowVideoBase : public CGUIMediaWindow, public IBackgroundLoaderObse
   bool OnMessage(CGUIMessage& message) override;
   bool OnAction(const CAction &action) override;
 
-  virtual void OnItemInfo(const CFileItem& fileItem, ADDON::ScraperPtr& scraper);
-
+  virtual bool OnItemInfo(const CFileItem& fileItem, ADDON::ScraperPtr& scraper);
```

Comment:
Maybe add docs to this method? What the bool is expected to represent?

Reply:
Fair. Will add some docs.

Reply:
@enen92 done.